src/System.Net.Sockets/src/System/Net/Sockets/BaseOverlappedAsyncResult.Unix.cs

```diff
@@ -23,7 +23,7 @@ public BaseOverlappedAsyncResult(Socket socket, Object asyncState, AsyncCallback
             if (NetEventSource.IsEnabled) NetEventSource.Info(this, socket);
         }
 
-        protected void CompletionCallback(int numBytes, SocketError errorCode)
+        internal void CompletionCallback(int numBytes, SocketError errorCode)
```

Comment:
Yes.

At some point we should try to get rid of the need for these partial AsyncResult classes, but until then we should just keep the existing pattern.